### PR TITLE
Command/bluemix

### PR DESCRIPTION
**Pull request by hacksparrow**
### Description

Added `bluemix` command

#### Related issues

- None

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: templates/projects/empty-server/files/server/server.js [hunk 1]
```diff
@@ -22,8 +21,8 @@ app.start = function() {
 // Sub-apps like REST API are mounted via boot scripts.
 boot(app, __dirname, function(err) {
   if (err) throw err;
-
   // start the server if `$ node server.js`
-  if (require.main === module)
-    app.start();
+  if (require.main === module) {
+    var server = app.start();
```
Comment (raymondfeng) [inline templates/projects/empty-server/files/server/server.js]: Why do we assign the return value to `server`, which is not referenced elsewhere?

File: package.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "morgan": "^1.2.0",
     "ncp": "^2.0.0",
     "normalize-git-url": "^3.0.2",
+    "request": "^2.81.0",
```
Comment (raymondfeng) [inline package.json]: Do we still need `request` as it's handled by `loopback-bluemix`?

**Comment by raymondfeng** [comment]
@hacksparrow Based on my observation, the PR doesn't add anything to `loopback-workspace` any more. Can we close it?

File: templates/projects/empty-server/files/server/server.js (hunk 1 above)
Comment (hacksparrow) [inline templates/projects/empty-server/files/server/server.js]: @raymondfeng appmetrics needs its reference, but I think we can handle it via `loopback-bluemix`.

**Comment by hacksparrow** [comment]
@raymondfeng closing this.

**Pull request closed without merging**